src/runtime/manifest.ts

```diff
@@ -1052,11 +1061,13 @@ ${e.message}
         }
 
         if (targetParticle) {
-          let targetConnection;
-          if (connectionItem.target.param) {
-            targetConnection = targetParticle.connections[connectionItem.target.param];
+          let targetConnection: HandleConnection;
+
+          // TODO(lindner): replaced param with name since param is not defined, but name/particle are...
```

Comment:
Looks like this was last touched by @dstockwell back in 2017.  Checking the PEG grammar/ast types it appears that this was incorrect.  Thanks @csilvestrini

Reply:
Good spotting! I don't know how you figured that out... this code is so hard to follow without types...